src/applications/pensions/containers/ConfirmationPage.jsx

```diff
@@ -33,77 +27,124 @@ class ConfirmationPage extends React.Component {
 
     return (
       <div>
-        <h3 className="confirmation-page-title">Claim submitted</h3>
-        <p>
-          We process claims in the order we receive them. Please print this page
-          for your records.
-        </p>
-        <p>We may contact you for more information or documents.</p>
-        <div className="inset">
-          <h4>
-            Veterans Pension Benefit Claim{' '}
-            <span className="additional">(Form 21P-527EZ)</span>
-          </h4>
-          <span>for {fullName}</span>
-
-          <ul className="claim-list">
-            <li>
-              <strong>Date submitted</strong>
-              <br />
-              <span>{submittedAt}</span>
-            </li>
-            {response?.confirmationNumber && (
-              <li>
-                <strong>Confirmation number</strong>
-                <br />
-                <span>{response?.confirmationNumber}</span>
-              </li>
-            )}
-            <li>
-              <strong>Pension Management Center</strong>
+        <h2>Your Veteran's Pension application</h2>
+
+        <va-alert status="success">
+          <h3>Thank you for submitting your Veterans Pension application.</h3>
+          <p>
+            We've received your Veterans Pension application (VA Form
+            21P-527EZ). After we complete our review, we'll mail you a decision
+            letter with the details of our decision.
+          </p>
+        </va-alert>
+
+        <va-summary-box>
+          <h3>Your information for this application</h3>
```

Comment:
We can use the v3 version of this component too and improve layout with a utility class.
```suggestion
         <va-summary-box uswds>
          <h3 slot="headline" className="vads-u-margin-top--0">
            Your information for this application
          </h3>
```